### PR TITLE
Convert Filters to use DropDownButton

### LINKED ISSUE #148: Update filters to use dropdown button

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Fixed some bad styling on the views saved modal
Added an option to not format the dropdwon button contents

Fixes #148

change-type: minor

### CODE REVIEW

File: src/components/Filters/ViewsMenu.js [hunk 1]
```diff
@@ -44,12 +44,14 @@ const ViewListItem = styled.li`
   }
   & > button {
     position: absolute;
-    top: 7px;
-    right: 10px;
+    top: 50%;
+    transform: translateY(-50%);
```
Comment (thgreasi) [inline src/components/Filters/ViewsMenu.js]: NIT: I would love to see some flexbox magic but works for me as well.

File: src/components/Filters/ViewsMenu.js [hunk 2]
```diff
@@ -89,33 +83,27 @@ class ViewsMenu extends Component {
...
-            left
-          />
-        )}
-        {this.state.showViewsMenu && (
-          <MenuPanel className='views-menu__panel'>
+          <Box py={1}>
             {!hasViews && (
-              <Box p={3}>{"You haven't created any views yet"}</Box>
+              <Box py={2} px={3}>
+                {"You haven't created any views yet"}
```
Comment (thgreasi) [inline src/components/Filters/ViewsMenu.js]: Super-NIT: How about backticks? I though that react devs hate double quotes :confused:

Comment (LucianBuzzo) [reply src/components/Filters/ViewsMenu.js]: This is auto-formatted by prettier, but maybe I can nudge it

Comment (thgreasi) [reply src/components/Filters/ViewsMenu.js]: No worries, forget it :+1:

**Pull request merged**